app/api/config.js

```diff
@@ -0,0 +1,9 @@
+﻿
+let config = {}
+
+config.api      = {};
+config.api.url  = process.env.API_URL || 'https://api.cbddev.xyz:443';
+
+config.encryptionPassword   = process.env.ENCRYPTION_PASSWORD
```

Comment:
can you throw an exception is missing or sort

Reply:
i am throwing exception in the encryption file